src/variables.scss

```diff
@@ -0,0 +1,7 @@
+$card-border-color: #f3f3f3;
+$link-color: #060b35;
+$secondary-accent: #616070;
+$blue-accent: #00acdc;
```

Comment:
Variable name should show their purpose, if this color in palette changes to red, you have to rename it in all places in your cases